hack/make-rules/verify.sh

```diff
@@ -138,9 +138,9 @@ FAILED_TESTS=()
 
 function print-failed-tests {
   echo -e "========================"
-  echo -e "${color_red}FAILED TESTS${color_norm}"
+  echo -e "${color_red:?}FAILED TESTS${color_norm:?}"
```

Comment:
This should be coming in from `lib/util.sh`, I would rather we understand why that's not getting sourced in such that these are visible, rather than defend against them being unassigned